ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java

```diff
@@ -117,12 +115,11 @@ public static UnsignedLong get_previous_total_balance(BeaconState state) {
   public static List<Integer> get_attesting_indices(
       BeaconState state, List<PendingAttestation> attestations) throws IllegalArgumentException {
 
-    HashSet<Integer> output = new HashSet<>();
+    TreeSet<Integer> output = new TreeSet<>();
     for (PendingAttestation a : attestations) {
       output.addAll(get_attestation_participants(state, a.getData(), a.getAggregation_bitfield()));
     }
     List<Integer> output_list = new ArrayList<>(output);
-    Collections.sort(output_list);
     return output_list;
   }
```

Comment:
This is a good fix. Even though they are roughly equivalent ... As the number of validators increases, this should become even more efficient.